daemons/gptp/common/avbts_port.hpp

```diff
@@ -237,6 +237,9 @@ typedef struct {
 	/* Set to true if the port is the grandmaster. Used for fixed GM in the the AVnu automotive profile */
 	bool testMode;
 
+	/* Set to true if the IFF_RUNNING flag for port is up. Used to filter false LINKUP/LINKDOWN events */
```

Comment:
IFF_RUNNING is Linux specific, correct? This is common port code. Correct comment should be something like "Set to true is the port's network interface reports the link is "up"".